main/src/cgeo/geocaching/ui/dialog/SimpleDialog.java

```diff
@@ -76,6 +76,7 @@
     private TextParam positiveButton = TextParam.id(android.R.string.ok);
     private TextParam negativeButton = TextParam.id(android.R.string.cancel);
     private TextParam neutralButton = TextParam.id(android.R.string.untitled);
+    private boolean neutralButtonNeedsSelection = true;
```

Comment:
otherwise the neutral button was not shown / was enabled only, if an item was selected